app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java

```diff
@@ -313,6 +317,17 @@ public void onDestroyView() {
     }
 
     @Override
+    public void onDetach() {
+        if (call == null) {
+            return;
```

Comment:
We always need to call super's onDetatch(). I would expect this to cause an error. Why arent you calling it first?